apis/openstack-nova/src/main/java/org/jclouds/openstack/nova/v2_0/handlers/NovaErrorHandler.java

```diff
@@ -97,6 +97,13 @@ else if (message.indexOf("already exists") != -1)
                exception = new ResourceNotFoundException(message, exception);
             }
             break;
+        case 500:
```

Comment:
it is part of the same fix, when floating IP are finished, openstack returns 500 instead of https://github.com/andreaturli/jclouds/blob/fix/openstack-nova-floating-ips/apis/openstack-nova/src/main/java/org/jclouds/openstack/nova/v2_0/handlers/NovaErrorHandler.java#L84 so I added that case to trigger the rollback cleanly.